scripts/coverage.js

```diff
@@ -10,7 +10,7 @@ var Mocha = require('mocha'),
     Instrumenter = require('istanbul').Instrumenter,
     Report = require('istanbul').Report,
     Collector = new require('istanbul').Collector,
-    sourcefiles = ['index.js', 'extensions.js', 'render.js'],
```

Comment:
No problem, the setup probably needs to be looked at, this is just getting it running again